docs/source/grpo_trainer.md

```diff
@@ -36,7 +37,13 @@ dataset = load_dataset("trl-lib/tldr", split="train")
 def reward_len(completions, **kwargs):
     return [abs(20 - len(completion)) for completion in completions]
 
-training_args = GRPOConfig(output_dir="Qwen2-0.5B-GRPO", logging_steps=10)
+training_args = GRPOConfig(
+    output_dir="Qwen2-0.5B-GRPO",
+    logging_steps=10,
+    gradient_accumulation_steps=16,
+    num_generations=8,
+    per_device_train_batch_size=8,
+)
```

Comment:
let's keep the default values here (I'm aiming for the most simple example possible)

Also, per_device_train_batch_size has a new default to avoid OOM